### ISSUE TITLE
Design tour with reduced steps & dialogue

### ISSUE THREAD

**Issue by Benjaki2** [nasa-gibs/worldview#102]
Reduce tour steps and dialogue so that the user can get going faster and add 'callouts' to assist the user while they are browsing on their own.

**Comment by ZachTRice** [comment]
When we update this tour functionality, we should look at replacing the jquery joyride plugin with a library that is well-maintained. There are a few options out there now that would be better to use going forward.

**Comment by ZachTRice** [comment]
There is a mix of jquery dialog and jquery.joyride for the tour; these can probably be combined.

**Comment by ZachTRice** [comment]
Notes from 08/08/2018 Meeting on design [v0.2.0](https://zachtrice.github.io/worldview-tour/walkthrough/v0.2.0/index.html)

Slide 1 (Event Grid)
- Remove excess text throughout the design
- Box 1: Add Application Overview Text
- Have 5 Top events in additional boxes
- Make the First box a "Featured Event"
- Incorporate the 'Natural Event' icons into the design
- Also note, we could remove the get started text and/or move it to an additional window (appearing before the slide 1 design)
Other considerations:
- Show description text on hover?
- Shoe an animated GIF of the event on hover?

Slide 2 (More Event Info)
- Not needed. Getting into the tour is the most important next step

Slide 3 (Tour Window (Start))
- Remove table of contents
- Focus on the description
- Make left/right arrows more obvious
- Incorporate a "close" button into the design

Slide 4 (Tour Finished)
- Rework content in this window. Simplify / Remove text
- Add a link to more events of the same type (Link to historical events in the future?)

**Comment by ZachTRice** [comment]
Notes from 08/16/2018 Meeting on design [v0.3.0a](https://zachtrice.github.io/worldview-tour/walkthrough/v0.3.0a/index.html), [v0.3.0b](https://zachtrice.github.io/worldview-tour/walkthrough/v0.3.0b/index.html), [v0.3.0c](https://zachtrice.github.io/worldview-tour/walkthrough/v0.3.0c/index.html), [v0.3.0d interactive walkthrough](https://zachtrice.github.io/worldview-tour/walkthrough/v0.3.0d/index.html)

General Notes / Ideas:
- Using database to feed in events (in the future); follow-up suggestions - Look into using the status app to deliver the event list overview
- Technical: Think about ways to share the tour. Shareable link that starts at step 1 (incorporate a link at end of tour and just have a URL parameter to share out).

Tour Startup:
- Remove "Read More", move this to the Finished modal if need b.
- Keep icons but move to same side / make consistent
- Add indicator for row dropdown (arrows collapsed/open states)
- Remove content? - Or Make content slide up from bottom on hover
- Make entire image box clickable to start the event
- New background image for "Getting Started"
- Shrink up "Getting Started" box size & text
- Add more NASA, EOSDIS, type branding to "Getting Started" area

Other Startup layout considerations:
- Create an example that uses a carousel
- Remove all image backgrounds and just make it a list of accordions (Except the Getting Started area)
- Create 3x2 grids of boxes with logo in upper-right / color border, interesting image, title on bottom and text that slides up on hover

Tour Started Slides:
- Slide out the panel from the side when tour is initiated
- Animate the app state on initial load
- Re-incorporate an arrow or hand to show actions being taken to change the app state per each step

Finished Slide:
- Close side tour panel when Finished appears
- Remove background image from this window
- Keep content + buttons but tweak the button colors
- Add a share tour link

**Comment by ZachTRice** [comment]
Notes from 08/23/2018 Meeting on design [v0.4.0](https://zachtrice.github.io/worldview-tour/walkthrough/v0.4.0/index.html)

Startup page feedback:
- Everyone agreed "Getting Started" looks best with white background
- Most liked the Grid layout, but there was still some interest in exploring the carousel and maybe using the list view on mobile (if we decide to support mobile)
- We discussed a "Change view" button but the general consensus was that this was over-complicate the design
- Everyone liked the idea of animating the images on hover
- A Masonry view was suggested

Actions:
- Create an interactive box view 
- Explore/create box view mixed with carousel
- Create a Masonry view
- Add animations in interactive walkthrough

**Comment by ZachTRice** [comment]
Notes from 09/06/2018 Meeting on design v0.5.0 interactive walkthroughs ([v0.5.0-boxes](https://zachtrice.github.io/worldview-tour/walkthrough/v0.5.0-boxes/index.html), [v0.5.0-boxes-large](https://zachtrice.github.io/worldview-tour/walkthrough/v0.5.0-boxes-large/index.html), [v0.5.0-carousel](https://zachtrice.github.io/worldview-tour/walkthrough/v0.5.0-carousel/index.html), [v0.5.0-masonry](https://zachtrice.github.io/worldview-tour/walkthrough/v0.5.0-masonry/index.html))

TLDR: Boxes with a scrollbar was the winner with a few minor tweaks needed.

Start of tour:
- We all agreed on the "boxes design" with a scroll-bar and lazy-loading as we get more events. There was a mix on whether to have 6 or 9 boxes but we decided this can be based on viewport height.
- Drop the "View More Events" box

End of tour:
- Increase font-weight for legibility
- Button colors should be different; look at the earthdata UI docs. -- This will also be addressed later as there are multiple button color inconsistencies throughout the app and a color scheme needs to be defined.
- Change "More Events" to "More Stories" (or something similiar), so that we aren't bound to only showing event-based walkthroughs. ("More Walkthroughs" was also suggested but we did not like this terminology)

More Comments
- Option to ignore the tour when returning?
- Create an issue to investigate not being able to open WV GIFs in Photoshop
